src/epd_driver/epd_driver.c

```diff
@@ -342,3 +377,84 @@ enum EpdDrawError epd_draw_image(EpdRect area, const uint8_t *data, const EpdWav
     };
     return epd_draw_base(area, data, no_crop, EPD_MODE_DEFAULT, temperature, NULL, waveform);
 }
+
+void epd_set_rotation(enum EpdRotation rotation) {
+      display_rotation = rotation;
+}
+
+uint8_t epd_get_rotation() {
+    return display_rotation;
+}
+
+uint16_t epd_rotated_display_width() {
```

Comment:
I'd just use int here, since C converts to int for calculations anyway and on the esp32 a 16 bit int is not more efficient than a 32 bit int.

Reply:
Updated in [last commit](https://github.com/martinberlin/epdiy-rotation/commit/bde584b9dd0dd0eb401f6a845eb9cafcdda8d744). Thanks for the detailed and clear review after my update!